src/pages/ResearcherConsole.js

```diff
@@ -152,17 +112,27 @@ export default function ResearcherConsole() {
     }
   };
 
+
+  //Draft delete, by referenceIds
+  const deleteDraftById = async ({ referenceId, collectionsClone }) => {
+    await DAR.deleteDar(referenceId);
+    const targetIndex = findIndex((draft) => {
+      return draft.referenceId === referenceId;
+    })(collectionsClone);
+
+    return targetIndex;
+  };
+
   //Draft delete, passed down to draft table to be used with delete button
-  const deleteDraft = async ({ referenceId, identifier }) => {
+  const deleteDraft = async ({ referenceIds, identifier }) => {
```

Comment:
Change `identifier` to `darCode` to match field in collection summary (notification message was saying "undefined")